packages/venia-ui/lib/components/CheckoutPage/ItemsReview/itemsReviewFragments.gql.js

```diff
@@ -12,6 +12,7 @@ export const ItemsReviewFragment = gql`
                 thumbnail {
                     url
                 }
+                stock_status
```

Comment:
Yes, IIRC I was seeing errors where cache data was partially filled. I'm a bit confused though because I would just expect AC to make a network call to fulfill missing data. I'll look into this and see if I can repro the original error.